NumericUI/scripts/mods/NumericUI/HudElementDodgeCount.lua

```diff
@@ -110,6 +169,7 @@ HudElementDodgeCount.update = function(self, dt, t, ui_renderer, render_settings
 	-- Reset to empty in case we can't fill it in
 	self._widgets_by_name.dodge_count.content.text = ""
 	self._widgets_by_name.debug_dodge_count.content.text = ""
+	self._widgets_by_name.dodge_timer.style.timer.size = {0, 0}
```

Comment:
we could not create this table every update
```suggestion
	local ZERO_SIZE = {0,0}
	....
	self._widgets_by_name.dodge_timer.style.timer.size = ZERO_SIZE
```